aws-cpp-sdk-transfer/include/aws/transfer/TransferClient.h

```diff
@@ -58,7 +59,7 @@ struct AWS_TRANSFER_API TransferClientConfiguration
 class AWS_TRANSFER_API TransferClient
 {
     public:
-        TransferClient(const std::shared_ptr<Aws::S3::S3Client>& s3Client, const TransferClientConfiguration& config);
+        TransferClient(const std::shared_ptr<Aws::S3::S3Client>& s3Client, const TransferClientConfiguration& config, const std::shared_ptr<Aws::Utils::Threading::BlockingExecutor>& executor = nullptr);
```

Comment:
I forgot about TransferClientConfiguration, I think executor should be moved to that.